src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1808,35 +1808,6 @@ public void LangVersion_ListLangVersions()
             }
         }
 
-
-        [Fact]
-        public void OptimizationLevelAdded_Canary()
```

Comment:
I wasn't sure why these tests were removed. Were they judged to be not useful?

Reply:
They weren't complete, they missed cases which is why we had the unit test rebuild bug. I moved them to the Rebuild unit test library and leveraged `CombinatorialData` to make sure all cases were covered.